src/screens/Lessons/screens/Lesson/components/LessonData/index.js

```diff
@@ -56,25 +60,49 @@ export default ({lesson}) => {
       title: videoTitleText,
       children: lesson.wistia_id
         ? <WistiaVideo wistiaId={lesson.wistia_id} />
-        : null
+        : isInstructorsOwnLesson 
+          ? <Anchor url={lesson.lesson_http_url}>
+              <Button type='primary'>
+                {uploadVideoTitleText}
+              </Button>
+            </Anchor>
+          : noVideoDescriptionText
     },
   ]
 
   return (
     <div>
-      {map(items, (item, index) => (
-        <div 
-          key={index}
-          className={`${index < (size(items) - 1) ? 'bb' : ''} pb3 mb3 b--gray`}
-        >
-          <Heading level='4'>
-            {item.title}
-          </Heading>
-          <div>
-            {item.children}
+
+      {isInstructorsOwnLesson 
+        ? <div className='mb3'>
+            <Anchor url={lesson.lesson_http_url}>
+              <Button 
+                type='warning'
+                size='small'
+                outline
+              >
+                {editLessonTitleText}
+              </Button>
+            </Anchor>
           </div>
-        </div>
-      ))}
```

Comment:
Adds a temporary "Edit" button that links to the lesson edit page on egghead.io, until we build a queue in the Instructor Center directly